support/ebpf/bpfdefs.h

```diff
@@ -103,39 +105,41 @@ static long (*bpf_probe_read_kernel)(void *dst, int size, const void *unsafe_ptr
       bpf_trace_printk(____fmt, sizeof(____fmt), ##__VA_ARGS__);                                   \
     })
 
-  #ifdef OPTI_DEBUG
-    #define DEBUG_PRINT(fmt, ...) printt(fmt, ##__VA_ARGS__);
-
-    // Sends `SIGTRAP` to the current task, killing it and capturing a coredump.
-    //
-    // Only use this in code paths that you expect to be hit by a very specific process that you
-    // intend to debug. Placing it into frequently taken code paths might otherwise take down
-    // important system processes like sshd or your window manager. For frequently taken cases,
-    // prefer using the `DEBUG_CAPTURE_COREDUMP_IF_TGID` macro.
-    //
-    // This macro requires linking against kernel headers >= 5.6.
-    #define DEBUG_CAPTURE_COREDUMP()                                                               \
-      ({                                                                                           \
+  #define DEBUG_PRINT(fmt, ...)                                                                    \
+    ({                                                                                             \
+      if (with_debug_output) {                                                                     \
```

Comment:
```suggestion
      if (__builtin_expect(with_debug_output, 0)) {                                                                     \
```
to enable branch predictor to assume debug is not enabled?